### PR TITLE
chore: add chromatic support

### PR DESCRIPTION
**Pull request by steve-haar**
add chromatic support

### CODE REVIEW

File: .github/workflows/pr-visual-regression.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: PR Visual Regression
+on:
+  workflow_run:
+    workflows: [PR Build]
+    types:
+      - completed
+jobs:
+  pr-preview:
+    name: PR Chromatic
```
Comment (kevinbuhmann) [inline .github/workflows/pr-visual-regression.yml:9]: These should match. I made the others match the file name.

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-visual-regression.yml (hunk 1 above)
Comment (steve-haar) [reply .github/workflows/pr-visual-regression.yml]: fixed

Comment (kevinbuhmann) [reply .github/workflows/pr-visual-regression.yml]: The `pr-preview` key is still the same.

Comment (steve-haar) [reply .github/workflows/pr-visual-regression.yml]: sorry, should be fixed now

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**